app/not-found.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+
+export default function NotFound() {
+  return (
+    <div className="flex h-screen w-screen items-center bg-gray-50">
+      <div className="container flex flex-col items-center justify-center px-5 text-gray-700 md:flex-row">
+        <div className="mx-8 w-full text-center lg:w-1/2">
+          <div className="font-dark mb-8  text-7xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Oops!</div>
+          <div className="flex justify-center">
+            <Image src="/Frame.png" layout="responsive" width={300} height={300} alt="404 Error" />
+          </div>
+          <div className="font-dark mb-8 mt-8  text-4xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Page not found</div>
```

Comment:
- `mb-8 mt-8` can be replaced by `my-8`
```suggestion
          <div className="font-dark my-8  text-4xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Page not found</div>
```
- what is the use of `font-dark`